proto/storage/api_token.proto

```diff
@@ -13,10 +13,11 @@ message TokenMetadata {
     string name                          = 2;
     repeated string roles                = 7;
     google.protobuf.Timestamp issued_at  = 4;
-    google.protobuf.Timestamp expiration = 5;
-    bool revoked                         = 6;
+    google.protobuf.Timestamp expiration = 5 [(gogoproto.moretags) = 'search:"Expiration,store"'];
+    bool revoked                         = 6 [(gogoproto.moretags) = 'search:"Expiration,store"'];
 
 
     string role = 3 [deprecated = true];
+    // Next available tag: 8
```

Comment:
nit: we usually do this at the _start_ of the message, so please move it to l.10.

Reply:
I'd rather not have this comment be the type description.
I'll move it to the start of the message, in the message body, separated from the fields by a blank line (an alternative is to not add it in the first place, as the message is not that big yet).

Reply:
![Screenshot 2023-03-01 at 22 47 01](https://user-images.githubusercontent.com/91869377/222272296-70599ca7-068f-4144-a700-cf81bc6d5ac4.png)
Moving the log to l.10 would result in that entry in the swagger file, which I find a bit misleading.